pod/perlop.pod

```diff
@@ -2862,6 +2859,52 @@ on the terminating line.
 If the terminating string is quoted, the type of quotes used determine
 the treatment of the text.
 
+ my $person = 'John';
+
+ print uc << "EOT";
+ Hello, $person!  
+ And the text goes on.
+ EOT
+
+This yields
+
+ HELLO, JOHN!  
+ AND THE TEXT GOES ON.
+
+The same result happens if the double quotes are omitted 
+
+ my $person = 'John';
+ print uc <<EOT;
+ Hello, $person!  
+ And the text goes on.
+ EOT
+
+ HELLO, JOHN!  
+ AND THE TEXT GOES ON.
+
+If instead, single quotes are used, we get
+
+ print uc <<'EOT';
+ Hello, $person!  
+ And the text goes on.
+ EOT
+
+ my $person = 'John';
+ HELLO, $PERSON!  
+ AND THE TEXT GOES ON.
+
+You can do fancy things like call a function with the here-document
```

Comment:
Suggested phrasing: "like use the here-document as an argument to a function."

Reply:
Though this is maybe out of place since the preceding examples already do this.

Reply:
A bit of grammatical pedantry here: `s/like use/such as use/`.

Reply:
looking further down in the pod, I realize it needs much more work